Greek Translation

pankgeorg opened the pull request:
# Notes:
I've kept the language as simple as possible, tried to use terms as used by [Prof Tsiodras](https://www.nytimes.com/2020/04/05/world/europe/scientists-coronavirus-heroes.html), added an extra reference to him in the valve mask argument and tried to reference greek wikipedia in the 1 case it was possible.

I looked at Twitter photos to assume the gender of people when I had to, but please let me know to update if. 
```
"ο" -> he
"η" -> she
```
Added the 'Tinos' font, because Raleway doesn't support greek.

pankgeorg commented:
@ismarou I've added a PR, feel free to comment and/or send suggestions!

ismarou commented on index-el.html:
```diff
@@ -56,7 +56,7 @@ <h5>
 
     <p>
       <a href="https://www.preprints.org/manuscript/202004.0203/v1">Τώρα</a> <a href="https://link.springer.com/article/10.1007/s11606-020-06067-8">γνωρίζουμε</a> <a href="https://science.sciencemag.org/content/368/6498/1422">ότι</a> οι μάσκες έχουν μια σημαντική επίδραση στην επιβράδυνση της μετάδοσης του κορονοϊού.
```
I would change 'Τωρα γνωρίζουμε' with 'Γνωρίζουμε πλέον'

pankgeorg replied:
agreed

ismarou commented on index-el.html:
```diff
@@ -56,7 +56,7 @@ <h5>
+      Παρά τα στοιχεία, κάποιοι άνθρωποι διαφωνούν με τη χρήση τους, διότι θεωρούν ότι είναι μια προσωπική επιλογή και όχι ένα δημόσιο θέμα για την προστασία της υγείας όλων μας.
     </p>
 
     <p>
```
'για να σταματήσουμε μια επιδημια' ---> 'για να σταμαγτήσουμε μια πανδημία'

pankgeorg replied:
@ismarou This change is best to be proposed to the original english text, which uses only the term "epidemic". If you feel that's important I'd think it's best to open an issue on the English text. We'll translate the original as is then.

ismarou replied:
I get what you say. Opening the issue in the English text would cause the extra problem/need of adapting all other translations, previous to ours, so I don't know how viable it really is. Since you have done all the work, it's your call to choose if you wish to be exact in terms of translation or semantics.

ismarou commented on index-el.html:
```diff
@@ -567,7 +567,7 @@ <h2>Πως Σταματάμε μια Επιδημία</h2>
+        Σύμφωνα με ανάλυση των <a href="https://www.sciencedirect.com/science/article/pii/S2468042720300191">David Fisman, Amy Greer, και Ashleigh Tuite</a>,
         "η επίδραση των μασκών και άλλων προστατευτικών προσώπου στη μείωση της μετάδοσης του κορονοϊού είναι πιθανό να είναι μεγαλύτερη αν προβλεφθεί η διαθεσιμότητα τους σε μη-προνομιούχους πληθυσμούς".
       </span>
     </aside>
```
εκφράζοτνας ---> εκφράζοντας in the paragram 'Το Ανθρώπινο Κράτος'

ismarou replied:
paragraph*

pankgeorg replied:
fixed, thanks!

ismarou commented on index-el.html:
```diff
@@ -645,7 +645,7 @@ <h2>Το Ανθρώπινο Κόστος</h2>
     <p>
-      Αν αρκετοί άνθρωποι φορέσουν μάσκες, μπορούμε να φτάσουμε το τέλος του λόγου, όπου η πιθανότητα μόλυνσης είναι μηδενική.
+      Αν αρκετοί άνθρωποι φορέσουν μάσκες, μπορούμε να φτάσουμε το τέλος του λόφου, όπου η πιθανότητα μόλυνσης είναι μηδενική.
       Έτσι οι μάσκες μπορούν να σταματήσουν μια επιδημία.
```
In general, I would change every 'επιδημία' with the word 'πανδημία', as this is one of the strongest arguments of people who refuse to wear them

pankgeorg replied:
see comment above.

aatishb commented:
Thanks @pankgeorg for working in this & @ismarou for reviewing! If it’s fine with you, I’ll go ahead and merge the latest changes into this branch, and will let you know if there’s anything left to do.

aatishb commented on index-el.html:
```diff
@@ -0,0 +1,748 @@
+      </label>
+
+      <graph :traces="graph3Traces(
+        '% Infected'
```
Could you also translate this? (This is for the hover text in the second graph)

aatishb commented:
Hi @pankgeorg, Thanks for your efforts on this. I merged the recent updates with this branch to bring it closer to merging. Here are the remaining steps:

- [ ] translate '% Infected'
- [ ] (optional) you're encouraged to add a line of credit & link to yourself for doing the translation, after the first sentence in the Credits/References 

One last thing I wanted to check is if Greek doesn't have gender neutral pronouns or articles for people, it would be good to use a roughly equal mix of male / female pronouns/articles, or even a greater fraction of female pronouns/articles, for places where the gender is unspecified (e.g. 'the contagious person'). I just want to make sure we don't use male pronouns/articles as the default when gender is unspecified. (It seems you may have already looked out for this?)

Thanks!

aatishb commented:
Tagging https://github.com/aatishb/maskmath/issues/4

pankgeorg commented:
@aatishb

- [x] Translated the `% infected`
- [x] Fixed some typos
- [x] Tried to gender-neutralize the text

In most of the text, I'm using `ο άνθρωπος` which is `the human being`, which unfortunately is male in Greek.
The status quo is to omit the `άνθρωπος` `[*]` and just refer to everyone as `ο`. I've added all instances of `άνθρωπος` where I'm using the male gender.

I'm also using `κάποιος` which means `someone`, which is intrinsicly male `(he/him)` and has the meaning of `some (hu)man` but the `άνθρωπος` is omitted (due to `[*]`) . In gender neutral texts this is expanded abbreviating the female version of `someone` to `κάποιος/α`, which i don't like because it is like 'default male 7 letters /i remembered female + 1`. I though that just `κάποιος` (with the omitted meaning of `άνθρωπος` was good enough but, on second though, I'm going with `κάποια ή κάποιος`, which is `someone (she/her) or someone (he/him)` which sounds the least weird and communicates the message.

I've pushed the respective branches, so if you're ok with this approach, you can proceed with the merge.

Thanks,
Panagiotis

pankgeorg commented:
Also, on an offtopic note, I've also translated the [video](https://www.youtube.com/watch?v=Y47t9qLc9I4) but it doesn't show up (I think it needs some approval from minutephysics?) Do you know what process I have to follow/if/how can I contact Henry?

aatishb reviewed (approved):
Thanks @pankgeorg, I'll do one last test and proceed to merge.

aatishb commented:
@pankgeorg re: video translation, I'll pass that on to Henry. (and thanks!) He's reachable via this page: https://www.minutephysics.com/contact

aatishb commented:
@all-contributors add @pankgeorg for translation

aatishb commented:
@all-contributors add @ismarou for review

aatishb commented:
Thanks @pankgeorg @ismarou for working on this! Merged.
